shared/dev/dumb-sheet.render.native.js

```diff
@@ -40,32 +37,33 @@ class Render extends Component<void, Props, any> {
     }, 0)
   }
 
-  _continueTest () {
-    if (isTesting && this.state.testIndex !== -1) {
+  _increment () {
+    if (this.props.autoIncrement && this.state.testIndex !== -1) {
       const {componentsOnly} = this._getComponents()
       if (this.state.testIndex >= componentsOnly.length) {
         this.setState({testIndex: -1})
       } else {
-        this.setState({testIndex: this.state.testIndex + TEST_RENDER_NUMBER})
+        this.setState({testIndex: this.state.testIndex + 1})
       }
     }
   }
 
   componentDidMount () {
-    if (isTesting) {
-      this._continueTest()
+    if (this.props.autoIncrement) {
+      this._increment()
     }
   }
 
   componentDidUpdate () {
-    setTimeout(() => {
-      this._continueTest()
-    }, 1)
+    if (this.props.autoIncrement) {
+      setTimeout(() => {
+        this._increment()
+      }, 1)
```

Comment:
I have some more commits for this tomorrow. Playing with doing a different
build without the packager (release build) and skipping things like the
profile loading / avatar images (like we do with screenshotting). Hoping to
make it more stable. Might be impossible before rn 32.1.

On Wednesday, August 31, 2016, Max Goodman <notifications@github.com> wrote:

> In shared/dev/dumb-sheet.render.native.js
> <https://github.com/keybase/client/pull/4053#discussion_r77090845>:
>
> >      }
> >    }
> >
> >    componentDidUpdate () {
> > -    setTimeout(() => {
> > -      this._continueTest()
> > -    }, 1)
> > +    if (this.props.autoIncrement) {
> > +      setTimeout(() => {
> > +        this._increment()
> > +      }, 1)
>
> Understood. 👍 with that feedback.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/4053/files/dc32ebab67e2b5011be080959d5dc7301771135f..a2530d5f598003cb6416f7a8cff641c1c3ab9514#r77090845>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAfhU3S5VFTZ6bG486FY-COdq9aqm3DJks5qlgs4gaJpZM4Jtgrl>
> .
>